src/main/java/JimmyException.java

```diff
@@ -0,0 +1,11 @@
+public class JimmyException extends Exception {
+
+    /**
+     *
+     */
+    private static final long serialVersionUID = 1L;
```

Comment:
maybe it's better use uppercase and underscore to name a constant value?

Reply:
I'll look into this, thanks.